dipu/torch_dipu/csrc_dipu/vendor/cuda/CudaGeneratorImpl.cpp

```diff
@@ -5,14 +5,14 @@
 
 namespace dipu {
 
-static const size_t states_size = 200 * sizeof(4120);
+static const size_t states_size = 200 * sizeof(decltype(4120));
 static const size_t seed_size = sizeof(uint64_t);
 static const size_t offset_size = sizeof(int64_t);
 static const size_t total_size = states_size + seed_size + offset_size;
```

Comment:
```suggestion
static constexpr size_t kStatesSize = 200 * sizeof(decltype(4120));
static constexpr size_t kSeedSize = sizeof(uint64_t);
static constexpr size_t kOffsetSize = sizeof(int64_t);
static constexpr size_t kTotalSize = states_size + seed_size + offset_size;
```
原命名对下面代码的理解产生了影响
最好能移入类里，方便理解
4120 还是显得很奇怪

Reply:
我倾向于不改，而且state看起来是个过时的东西了
代码基本都是从这里来的 https://github.com/pytorch/pytorch/blob/v2.0.0/aten/src/ATen/cuda/CUDAGeneratorImpl.cpp 我本来想都nolint的